src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+
+    >>> swallow_square_brackets("qui aliquod institui[t] exemplum")
+    'qui aliquod institui exemplum'
+
+    >>> swallow_square_brackets("posthac tamen cum haec [tamen] quaeremus,")
+    'posthac tamen cum haec  quaeremus,'
+
+    """
+    return swallow(text, SQUARE_BRACKETS)
+
+
+def swallow_obelized_words(text: str) -> str:
+    """
+    Swallow obelized words; handles enclosed and words flagged on the left.
+    Considers plus signs and daggers as obelization markers
+    :param text: Text with obelized words
+    :return: clean text
+
+    >>> swallow_obelized_words("tu Fauonium †asinium† dicas")
+    'tu Fauonium  dicas'
+
+    >>> swallow_obelized_words("tu Fauonium †asinium dicas")
+    'tu Fauonium dicas'
+
+    >>> swallow_obelized_words("meam +similitudinem+")
+    'meam'
+
+    >>> swallow_obelized_words("mea +ratio non habet" )
+    'mea non habet'
+
+    """
+    text = swallow(text, OBELIZED_WORDS)
+    text = swallow(text, OBELIZED_WORD)
+    text = swallow(text, OBELIZED_PLUS_WORDS)
+    return swallow(text, OBELIZED_PLUS_WORD)
+
+
+def disappear_round_brackets(text: str) -> str:
+    """
+    Remove round brackets and keep the text intact
+    :param text: Text with round brackets.
+    :return: Clean text.
+
+    >>> disappear_round_brackets("trib(unus) mil(itum) leg(ionis) III")
+    'tribunus militum legionis III'
+    """
+    text = text.replace("(", "")
+    return text.replace(")", "")
+
+
+def swallow_editorial(text: str) -> str:
+    """
+    Swallow common editorial morks
+    :param text: Text with editorial marks
+    :return: Clean text.
+
+    >>> swallow_editorial("{PRO P. QVINCTIO ORATIO} Quae res in civitate trib(unus) mil(itum) leg(ionis) III tu Fauonium †asinium† dicas meam +similitudinem+  mea +ratio non habet ...     ")
+    '{PRO P. QVINCTIO ORATIO} Quae res in civitate tribunus militum legionis III tu Fauonium  dicas meam   mea non habet ...'
+
+    """
+    text = disappear_round_brackets(text)
+    text = swallow_angle_brackets(text)
+    text = swallow_square_brackets(text)
+    text = swallow_obelized_words(text)
+    return text
+
+
+def accept_editorial(text: str) -> str:
+    """
+    Accept common editorial suggestions
+    :param text: Text with editorial suggestions
+    :return: clean text
+
+    >>> accept_editorial("{PRO P. QVINCTIO ORATIO} Quae res in civitate trib(unus) mil(itum) leg(ionis) III tu Fauonium †asinium† dicas meam +similitudinem+  mea +ratio non habet ...     ")
+    'Quae res in civitate tribunus militum legionis III tu Fauonium  dicas meam   mea non habet  '
+
+    """
+    text = swallow_braces(text)
+    text = disappear_round_brackets(text)
+    text = swallow_obelized_words(text)
+    text = text.replace("[", "")
+    text = text.replace("]", "")
+    text = text.replace("<", "")
+    text = text.replace(">", "")
+    text = text.replace("...", " ")
+    return text
+
+
+def truecase(word: str, case_counter: Dict[str, int]):
```

Comment:
This is fabulous; users will certainly need to see good tutorial illustrating how/when to do this.